exercises/poker/poker_test.exs

```diff
@@ -0,0 +1,215 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("poker.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule PokerTest do
+  use ExUnit.Case
+
+  #@tag :pending
+  test "single hand always wins" do
+    high_of_jack = ~w(4S 5S 7H 8D JC)
+    assert [high_of_jack] == Poker.best_hand([high_of_jack])
```

Comment:
It is the common convention to put the actual output on the left and the expected result on the right. Please reverse the tests to match. Thanks,